README.md

```diff
@@ -1,96 +1,117 @@
 ![Latest GitHub Release](https://img.shields.io/github/v/release/byu-oit/terraform-aws-postman-test-lambda?sort=semver)
 
 # Terraform AWS Postman Test Lambda
+
 Terraform module that creates a generic lambda function that runs newman tests against a postman collection.
 
-This lambda function is intended for use with [CodeDeploy's lifecycle hooks](https://docs.aws.amazon.com/codedeploy/latest/userguide/reference-appspec-file-structure-hooks.html).
-This lambda function will attempt to run the [newman](https://www.npmjs.com/package/newman) CLI to run your Postman collection as a test.
-This lambda function will tell CodeDeploy if the tests pass or fail.
+This lambda function is intended for use
+with [CodeDeploy's lifecycle hooks](https://docs.aws.amazon.com/codedeploy/latest/userguide/reference-appspec-file-structure-hooks.html)
+. This lambda function will attempt to run the [newman](https://www.npmjs.com/package/newman) CLI to run your Postman
+collection as a test. This lambda function will tell CodeDeploy if the tests pass or fail.
 
 #### [New to Terraform Modules at BYU?](https://github.com/byu-oit/terraform-documentation)
 
 ## Usage
-You can provide a postman collection and environment to be tested in one of two ways:
-1. Provided in your github repo
-    ```hcl
-    module "postman_test_lambda" {
-      source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.4.0"
-        app_name                      = "simple-example"
-        postman_collection_file       = "terraform-aws-postman-test-lambda-example.postman_collection.json"
-        postman_environment_file      = "terraform-aws-postman-test-lambda-env.postman_environment.json"
-        role_permissions_boundary_arn = data.aws_ssm_parameter.role_permissions_boundary_arn.value
-    }
-    ```
-2. Or from the Postman API
-    ```hcl
-    module "postman_test_lambda" {
-      source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.4.0"
-        app_name                      = "simple-example"
-        postman_collection_name       = "terraform-aws-postman-test-lambda-example"
-        postman_environment_name      = "terraform-aws-postman-test-lambda-env"
-        postman_api_key               = var.postman_api_key
-        role_permissions_boundary_arn = data.aws_ssm_parameter.role_permissions_boundary_arn.value
+
+```hcl
```

Comment:
I know we include it in the examples but do you think it would be worth it to still include how to pass collections by UID here?

Reply:
Good idea. I updated it to have the example in the Using Postman API section in the readme